sdk/src/derivation_path.rs

```diff
@@ -79,7 +79,7 @@ impl DerivationPath {
         Ok(Self::new_bip44_with_coin(coin, account, change))
     }
 
-    fn from_absolute_path_str(path: &str) -> Result<Self, DerivationPathError> {
+    pub fn from_absolute_path_str(path: &str) -> Result<Self, DerivationPathError> {
```

Comment:
This will be unnecessary with the switch to `SignerUri`

Reply:
For `grind` I need var with DerivationPath in this scope.
But all helpers in clap-utils/keypair.rs get "context" variables, not simple values. And they have cli output. But it isn't right for `grind`.
The best what I find is `pub(crate) pub fn parse_signer_source<S: AsRef<str>>` in clap-utils/keypair.rs. But it is private for me too.
So I need to write one more helper in clap-utils/keypair.rs for `grind` with derivation path. Or where am I wrong? Could you point me the right way you mean?

Reply:
Assuming you're referring to `fn parse_signer_source`, @t-nelson , totally agree. Fine to make fully `pub`.

Reply:
> Assuming you're referring to `fn parse_signer_source`, @t-nelson , totally agree. Fine to make fully `pub`.

Yep, exactly!  Sorry I wasn't more clear.  I need to not review PRs at midnight :sweat_smile:

Reply:
In fact, I began to use a new helper `derivation_path_from_path()` in `clap-utils` and `parse_signer_source()` in it. With `prompt://` this solution looks better.
So `from_absolute_path_str()` is private again now